docs/index.md

```diff
@@ -136,7 +136,7 @@ class Action < Granite::Action
   collection :ids, Integer
 
   private def execute_perform!(*)
-    puts "Hello #{name}! We have the following ids: #{ids}'
+    puts "Hello #{name}! We have the following ids: #{ids}"
```

Comment:
Eagle eye 👍

Reply:
Thank you for reviewing @pirj.

What's the protocol for merging in this project? 
Do I do it? 
If so, how many approvals are required?

🙇

Reply:
I guess we should expand on the contribution guidelines.